### PR TITLE
How to use the Azure CLI effectively | Reenter dropped code blocks

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/use-cli-effectively.md [hunk 1]
```diff
@@ -183,7 +278,7 @@ Some proxies require authentication. The format of the `HTTP_PROXY` or `HTTPS_PR
 
 If you run Azure CLI on a build machine where multiple jobs can be run in parallel, access tokens might be shared between two build jobs run as the same OS user.  To avoid mix ups, set `AZURE_CONFIG_DIR` to a directory where the access tokens are stored.
 
-## Generic update arguments
+## Generic update parameters
```
Comment (jiasli) [inline docs-ref-conceptual/use-cli-effectively.md:281]: In-tool helps uses the word `argument`. Any reason for the change?

```
> az group create -h

Command
    az group create : Create a new resource group.

Arguments
    --location -l                 [Required] : Location. Values from: `az account list-locations`.
                                               You can configure the default location using `az
                                               configure --defaults location=<location>`.
    --name --resource-group -g -n [Required] : Name of the new resource group.
    --managed-by                             : The ID of the resource that manages this resource
                                               group.
    --tags                                   : Space-separated tags: key[=value] [key[=value] ...].
                                               Use "" to clear existing tags.
```

Comment (dbradish-microsoft) [reply docs-ref-conceptual/use-cli-effectively.md:281]: @jiashli,  I don't know why the docs use `parameter` (a placeholder for a value) when the in-tool help uses `arguments` (the value passed), but conceptual and autogen docs should use the same word.  Even `az config param-persist` kept the word `parameter`.

Technicall, the `--location` parameter can contain the following `arguments`: eastus, westus

**Comment by dbradish-microsoft** [comment]
@jiasli , @chasewilson , This article is ready for your review.  The purpose of this PR is to replace removed information and code blocks.  I also reviewed many 2022 PRs to make sure those changes did not get overwritten by this update.  Existing open PRs that modify this same article may need to be updated to resolve conflicts.

**Comment by dbradish-microsoft** [comment]
@jiasli / @chasewilson , I am going to merge this PR in order to complete this outstanding task by end of the month / fiscal year.  This article will continue to change as we work with customers and seek to provide new tips and techniques for working with the Azure CLI.

**Pull request merged**